### ISSUE TITLE
Staging (testnet) is setting by default mainnet markets instead of testnet

### ISSUE THREAD

**Issue by sirsoth** [aave/interface#725]
**Describe the bug**
By default 'testnet mode' is disabled at https://staging.aave.com/ which makes the testing environment to only show mainnet markets.

**To Reproduce**
Access https://staging.aave.com/ and only mainnet markets are shown

**Expected behavior**
Testnet mode should be by default enabled, so testnet markets are shown, and best would be that the default market to load is one of the testnet ones too. 

**Screenshots**

![image](https://user-images.githubusercontent.com/5862694/165749066-6d50332e-a43b-4ba4-8608-af881b7ac8c0.png)

![image](https://user-images.githubusercontent.com/5862694/165749096-74403bac-e519-4384-bff1-776a28f4aa97.png)

**Additional context**
As fix I would just enabled testnet mode by default at staging, or if you consider you can also disable mainnet markets there, but I feel mainnet markets are not that bad as it might act as "secondary" interface if needed.

**Comment by sakulstra** [comment]
@sirsoth afaik, staging is no longer actively deployed as there's no intermediate staging any more.
Things merged to main are directly going to production - not 100% sure but would assume current staging is either outdated or just production.

**Comment by sirsoth** [comment]
So the idea from now on is that people test at app.aave.com just using the 'testnet mode' switch right? as the FAQ is currently referring to legacy testnet URLs (https://docs.aave.com/faq/testing-aave#introduction) just to get the information updated if so

**Comment by sakulstra** [comment]
I'm no longer at aave companies, so @sendra / @foodaka have the final say, but i'd say so.

If you think that it would be better to have a dedicated staging where testnets are default enabled that's possible as well but requires some work & not sure if worth it.

**Comment by sendra** [comment]
Testnet is not set by default. Lets Use this issue for Updating FAQs only

**Comment by sirsoth** [comment]
Yes @foodaka the proposal of setting by default testnet was based on the premise that there was a "testnet.aave.com" or similar URL, as the new approach is that testnet is accessible at app.aave.com then I agree, testnet mode should be disabled by default, and its only needed to update properly the FAQ info, as soon as its updated Im closing this issue, thanks guys

**Comment by grothem** [comment]
I started looking into this issue today. From my understanding, staging.aave.com will show test nets only, no mainnets. app.aave.com will not show any test nets. We'll need to:

- Remove the 'Test net mode' toggle from the settings menu
- Set the NEXT_PUBLIC_ENABLE_TESTNET env var to true for staging, and false for prod in the GitHub workflow
- Update the FAQ documentation

Does that sound right?

**Comment by defispartan** [comment]
> 
NEXT_PUBLIC_ENABLE_TESTNET is set to true for the staging workflow in this PR (https://github.com/aave/interface/commit/344fa996e3decbdca9efa1d44df4a9b9c82b489f) so that part's already done.

I would say let's keep the 'Testnet Mode' flag, but only for app.aave.com, so the changes would be:

- Hiding the production markets if *either* the NEXT_PUBLIC_ENABLE_TESTNET environment variable or localStorage.testnetsEnabled (set by the toggle) is set to true.

- Remove the 'Testnet Mode' toggle if NEXT_PUBLIC ENABLE_TESTNET is set to true

**Comment by grothem** [comment]
I was going to get a PR up for updating the FAQ, but thought I'd double check here first. Do we want to specifically call out staging.aave.com to use for testing? Right now the documentation is accurate, it just says to go to app.aave.com and toggle on testnet mode.
